packages/edit-site/src/components/page-actions/trash-page-menu-item.js

```diff
@@ -51,11 +51,7 @@ export default function TrashPageMenuItem( { postId, onRemove } ) {
 	}
 	return (
 		<>
-			<MenuItem
-				onClick={ () => removePage() }
-				isDestructive
-				variant="secondary"
-			>
+			<MenuItem onClick={ () => removePage() } isDestructive>
```

Comment:
Is this change meant to be part of this PR? I didn't see it mentioned in the description...